zds/tutorialv2/tests/tests_views/tests_published.py

```diff
@@ -1998,14 +1998,14 @@ def check_images_socials(self, result, pattern_link_image, title, description):
                 result.content.decode("utf-8"),
                 start_reg + meta + r'(\S+)(\s+)content="https://(\S+)/' + pattern_link_image + r'(\S+)"',
             )
-        for meta in ["twitter:description"]:
+        for meta in ["X:description"]:
```

Comment:
```suggestion
        for meta in ["twitter:description"]:
```

Le nom des paramètres `meta` [n’a pas changé](https://developer.x.com/en/docs/twitter-for-websites/cards/overview/markup).